src/language-js/print/flow.js

```diff
@@ -276,6 +277,18 @@ function printFlow(path, options, print) {
           ? ["(", print("value"), ")"]
           : []),
       ];
+
+    case "AsExpression":
+    case "AsConstExpression":
+    case "SatisfiesExpression": {
+      const operator =
+        node.type === "AsExpression"
+          ? "as"
+          : node.type === "AsConstExpression"
+          ? "as-const"
+          : "satisfies";
```

Comment:
Let's get `operator` in `printBinaryCastExpression`.

Reply:
So we can use `node.type === "AsExpression" || node.type === "TSAsExpression"`.

Reply:
I see thumbs up here, but code didn't change.

Reply:
Changes should be amended and force pushed now